src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -1298,14 +1330,10 @@ def number(precision: int = 38, scale: int = 0) -> str:
         + RIGHT_PARENTHESIS
     )
 
+
 def string(length: Optional[int] = None) -> str:
     if length:
-        return (
-            STRING
-            + LEFT_PARENTHESIS
-            + str(length)
-            + RIGHT_PARENTHESIS
-        )
+        return STRING + LEFT_PARENTHESIS + str(length) + RIGHT_PARENTHESIS
```

Comment:
Why was formatting changed?

Reply:
@sfc-gh-vnayak Is this pre-commit linter?

Reply:
This was due to black auto-formatting; it's part of the pre-commit hooks added to my setup during bootcamp. Looking into removing the hook.